cabot/cabotapp/migrations/0030_auto__chg_field_graphitestatuscheck_interval__chg_field_graphitestatus.py

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+
+        # Changing field 'GraphiteStatusCheck.interval'
+        db.alter_column(u'cabotapp_graphitestatuscheck', 'interval', self.gf('django.db.models.fields.PositiveIntegerField')())
```

Comment:
How does this migration work when there's data in the column? What if there's a negative number in the column (there shouldn't be but...)? Does it do anything to the data in the column?

If you haven't done this already you can try this locally if you migrate to 0029, then create some checks, then migrate to 0030